### PR TITLE
Restyle notifications

### PR DESCRIPTION
**Pull request by sparrowDom**
### Description:
Restyle notifications according to designs in Zeplin

**Related issue:** https://github.com/OriginProtocol/origin/issues/2455

### Checklist:

- [x] Test your work and double-check to confirm that you didn't break anything
- [ ] [Wrap any new text/strings with `fbt`](https://github.com/OriginProtocol/origin/tree/master/dapps/marketplace/translation#wrapping-text) for translation
- [ ] If there are any new/changed strings to translate, [`npm run translate`](https://github.com/OriginProtocol/origin/tree/master/dapps/marketplace/translation#integrating-translations)
- [ ] Update any relevant READMEs and [docs](https://github.com/OriginProtocol/origin/tree/master/docs)

### CODE REVIEW

File: dapps/marketplace/src/pages/notifications/NotificationRow.js [hunk 1]
```diff
@@ -14,8 +13,22 @@ const Row = ({ node, identity, onClick }) => {
...
+  const buyerId = get(node, 'offer.buyer.id')
+  const partyId = get(node, 'party.id')
+  const nameLink = <Link to={`/user/${partyId}`}>{name}</Link>
+
+  const partyRole =
+    partyId === buyerId
+      ? fbt('Buyer:', 'notifications.buyer')
+      : fbt('Seller:', 'notifications.seller')
 
+  const shortenWallet = wallet => {
```
Comment (nick) [inline dapps/marketplace/src/pages/notifications/NotificationRow.js]: Can use `utils/formatHash` here. Need to document all these util functions :)

File: dapps/marketplace/src/utils/distanceToNow.js [hunk 2]
```diff
@@ -1,9 +1,13 @@
-export default function distanceToNow(timestamp, long) {
+import { fbt } from 'fbt-runtime'
+
+export default function distanceToNow(timestamp, { long, showJustNow } = {}) {
```
Comment (nick) [inline dapps/marketplace/src/utils/distanceToNow.js]: There are a bunch of other places in the source that use this function... do we need to update them too?

Comment (sparrowDom) [reply dapps/marketplace/src/utils/distanceToNow.js]: I've done [a search](https://github.com/OriginProtocol/origin/search?q=distanceToNow&unscoped_q=distanceToNow) and none of them pass the 2nd argument.

File: dapps/marketplace/src/utils/distanceToNow.js [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
   const now = new Date().getTime()
   if (typeof timestamp === 'string') timestamp = Number(timestamp)
   const diff = now / 1000 - timestamp
   if (diff < 60) {
-    return `<1${long ? ' minute' : 'm'}`
+    return showJustNow
+      ? fbt('Just now', 'time.justNow')
+      : `<1${long ? ' minute' : 'm'}`
```
Comment (shahthepro) [inline dapps/marketplace/src/utils/distanceToNow.js]: Just a very minor nitpick, Shouldn't this be translated?

File: dapps/marketplace/src/pages/notifications/NotificationRow.js (hunk 1 above)
Comment (sparrowDom) [reply dapps/marketplace/src/pages/notifications/NotificationRow.js]: 👍

File: dapps/marketplace/src/utils/distanceToNow.js (hunk 3 above)
Comment (sparrowDom) [reply dapps/marketplace/src/utils/distanceToNow.js]: yup good catch 👍

**Comment by sparrowDom** [comment]
@shahthepro or @nick can any of you take another look please?

**Comment by sparrowDom** [comment]
thanks @nick

**Pull request merged**